github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -45,13 +45,18 @@ presubmits:
           value: "kind-1.19-sriov"
         - name: "KUBEVIRT_NUM_NODES"
           value: "3"
+        - name: "RUN_KUBEVIRT_CONFORMANCE"
+          value: "true"
+        - name: "SONOBUOY_EXTRA_ARGS"
+          value: "--plugin-env kubevirt-conformance.E2E_FOCUS=SRIOV"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
```

Comment:
worth to add please -e so it will fail in case of err

Reply:
The trap will catch it isnt it?

Reply:
Nope
```
$ ./try.sh 
done
teardown
```
```
#!/bin/bash

trap "echo teardown" EXIT SIGINT

false
echo done
```

It doesnt raise signal on failures without it

Reply:
ahhh because we dont have `&&` now
I will add it